extras/kms/repository.go

```diff
@@ -17,7 +17,8 @@ const (
 	// DefaultLimit is the default for results
 	DefaultLimit = 10000
 
-	DefaultWrapperSecret = "secret"
+	// DefaultWrapperSecret defines a default secret for testing
+	DefaultWrapperSecret = "secret1234567890"
```

Comment:
Do we need to define this in this package, vs putting it in the test file?

Reply:
it's used in both internal and external pkg tests.